src/xrGame/obstacles_query_inline.h

```diff
@@ -10,7 +10,13 @@
 #define OBSTACLES_QUERY_INLINE_H
 
 IC obstacles_query::obstacles_query() { init(); }
-IC void obstacles_query::init() { m_crc = 0; }
+
+IC void obstacles_query::init()
+{
+    m_crc = 0;
+    m_actual = true;
```

Comment:
Comment for future, if someone would investigate something:
`m_actual` wasn't set to any value and setting it to `true` here is correct – `m_obstacles` at empty at this point, so there's no point to call `compute_area()` function which `m_actual` is only used for.